hips/hip-0012.md

```diff
@@ -44,153 +147,117 @@ The person will have the following responsibilities:
-2. Feature development
-3. Release preparation
-
-The timeline will be approximately:
-
-- Naming of a Release Engineer: Sept. 2021
-- Kick-off meeting: Oct. 2021
-- Earliest that engineering can begin: Jan. 2022
-- Earliest that a release could possibly be: May, 2022
-
-More likely, the release will be late 2022 to midyear 2023.
+If the release engineer is unable to fulfill their duties, the Helm project maintainers may select a replacement.
 
 #### Planning
 
-This phase begins when the release engineer is named.
-The release engineer shall schedule a kick-off meeting in which all Helm maintainers (from any Helm org project) will be invited to participate.
-Beyond these participants, the release engineer may determine who else will be included, and may determine the meeting to be public.
-The kick-off meeting will open the floor to feature ideas and changes, with preference toward ideas already documented as HIPs.
-
-This process will continue as long as the release engineer determines necessary,
-but not less than three months.
-This minimum time frame is intended to give ample time for the community to provide input.
+This phase begins when the release engineer is appointed. The release engineer shall schedule a kick-off meeting, inviting all Helm maintainers from any Helm org project to attend.
+The release engineer may also decide who else to include and whether the meeting will be public.
+During the kick-off meeting, participants are encouraged to propose feature ideas and changes, with preference given to those already documented as HIPs.
 
-All features and breaking changes MUST be described in a HIP, and that HIP must pass the usual process.
-This provides three benefits:
+This process continues until the release engineer deems it sufficient, but for a minimum of three months, to allow ample community input.
 
+All proposed features and breaking changes must be described in a standard HIP or Helm 4 proposal.
+This approach offers several advantages:
 - All changes are documented
-- All changes have had a period of scrutiny prior to the PR being written
-- There is a clear record of alternative considerations
+- Changes undergo scrutiny before PR creation
+- A clear record of alternative considerations exists
 
-Things that are deemed by core maintainers to be "bug fixes" (and not changes or features) do not need HIPs.
+HIPs or proposals are not required for "bug fixes" or cleanup as determined by core maintainers.
 
-The outcome of the planning cycle is a clear record of approved HIPs describing Helm 4 features.
-This must be done prior to development beginning.
-This list of HIPs will serve as the "blueprint" for working on Helm 4.
+The outcome of the planning cycle is a well-documented list of approved HIPs detailing Helm 4's features, which must be completed before development commences.
+This list will serve as the foundation for working on Helm 4.
+
+Cleanup and non-functional changes may proceed while in the planning phase.
```

Comment:
If the planning phase is 3 months we will only end up with 4-5 months to code any changes. This is not a large enough window for development. I would like to suggest some revisions.

* Not all planning needs to be done up front. We have an initial planning phase but additions can come in later.
* Once something is approved, the work can begin on it. We do not need to wait.
* The planning phase ends January 6, 2025. This date is specific because it's when the RC for the next Helm minor release goes out. It will give us around 2 months for initial planning.

Thoughts?

Reply:
Reduced to two months and relaxed verbiage around changes being allowed before planning completes. But also included the rational for planning: that we do want to try and build the list of main changes prior to focusing on implementation